lib/srv/db/access_test.go

```diff
@@ -1950,6 +1955,20 @@ func (p *agentParams) setDefaults(c *testContext) {
 			},
 		}
 	}
+
+	if p.CloudClients == nil {
+		p.CloudClients = &clients.TestCloudClients{
+			STS:                &cloudtest.STSMock{},
```

Comment:
this can be wrapped in a `cloudtest.New` that fills the default values

Reply:
It's not very common to mock all the clients like this. So will keep this one as it is (being lazy) for now.